.github/workflows/docs.yml

```diff
@@ -0,0 +1,33 @@
+name: Build and deploy JavaDocs
+
+on:
+  push:
+    branches:
+      - dev
```

Comment:
Docs should only change when releasing a new version
```suggestion
      - master
```

Reply:
Changed 👍